### PR TITLE
Fiona 1598 properties get help text

### PR DESCRIPTION
**Pull request by Akyiaa**
Story https://github.com/galasa-dev/projectmanagement/issues/1598

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

File: docs/generated/errors-list.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ The `galasactl` tool can generate the following errors:
 - GAL1093E: Failed to submit test (bundle: {}, class: {}). Reason is: {}
 - GAL1094E: User error: Cannot run test {} on a local JVM because no OBR information is available. Supply an OBR using the --obr parameter, or (if using a portfolio) ensure the portfolio contains an OBR for this test.
 - GAL1095E: namespace '{}' is invalid. Expected it to be in a format starting with letters, and ending in a number with no non-alphanumeric characters.
-- GAL1096E: Could not query namespace results.
+- GAL1096E: Could not query namespace results. Reason: '{}
```
Comment (eamansour) [inline docs/generated/errors-list.md:99]: Reason is missing a closing ' :)

File: pkg/cmd/properties.go [hunk 2]
```diff
@@ -32,9 +32,9 @@ func init() {
 			"Example: http://example.com/bootstrap, file:///user/myuserid/.galasa/bootstrap.properties , file://C:/Users/myuserid/.galasa/bootstrap.properties")
 
 	cmd.PersistentFlags().StringVarP(&namespace, "namespace", "s", "",
-		"Namespace. A container for a collection of properties. "+
+		"Namespace. A mandatory flag that describes a container for a collection of properties. "+
```
Comment (KirbyKatcher) [inline pkg/cmd/properties.go:35]: Can we change this to `A mandatory flag that describes the container for a collection of properties.`

File: pkg/cmd/root.go [hunk 3]
```diff
@@ -132,11 +133,30 @@ func init() {
...
+	})
+
 	RootCmd.PersistentFlags().StringVarP(&CmdParamGalasaHomePath, "galasahome", "", "",
 		"Path to a folder where Galasa will read and write files and configuration settings. "+
 			"The default is '${HOME}/.galasa'. "+
 			"This overrides the GALASA_HOME environment variable which may be set instead.",
 	)
 }
+
+func Walk(c *cobra.Command, f func(*cobra.Command)) {
```
Comment (KirbyKatcher) [inline pkg/cmd/root.go:157]: not sure what f and c are variables of?

File: pkg/cmd/root.go [hunk 4]
```diff
@@ -132,11 +133,30 @@ func init() {
 			"An existing file will be overwritten. "+
 			"Specify \"-\" to log to stderr. "+
 			"Defaults to not logging.")
+
 	RootCmd.SetHelpCommand(&cobra.Command{Hidden: true})
 
+	Walk(RootCmd, func(c *cobra.Command) {
+		alias := c.NameAndAliases()
```
Comment (KirbyKatcher) [inline pkg/cmd/root.go:140]: please change c to command as a variable name

Comment (eamansour) [reply pkg/cmd/root.go:140]: +1

File: pkg/cmd/root.go (hunk 3 above)
Comment (eamansour) [reply pkg/cmd/root.go:157]: +1

File: pkg/errors/errorMessage.go [hunk 5]
```diff
@@ -180,7 +180,7 @@ var (
 	GALASA_ERROR_FAILED_TO_SUBMIT_TEST               = NewMessageType("GAL1093E: Failed to submit test (bundle: %s, class: %s). Reason is: %s", 1093, STACK_TRACE_NOT_WANTED)
 	GALASA_ERROR_NO_OBR_SPECIFIED_ON_INPUTS          = NewMessageType("GAL1094E: User error: Cannot run test %s on a local JVM because no OBR information is available. Supply an OBR using the --obr parameter, or (if using a portfolio) ensure the portfolio contains an OBR for this test.", 1094, STACK_TRACE_NOT_WANTED)
 	GALASA_ERROR_INVALID_NAMESPACE                   = NewMessageType("GAL1095E: namespace '%s' is invalid. Expected it to be in a format starting with letters, and ending in a number with no non-alphanumeric characters.", 1095, STACK_TRACE_NOT_WANTED)
-	GALASA_ERROR_QUERY_NAMESPACE_FAILED              = NewMessageType("GAL1096E: Could not query namespace results.", 1096, STACK_TRACE_WANTED)
+	GALASA_ERROR_QUERY_NAMESPACE_FAILED              = NewMessageType("GAL1096E: Could not query namespace results. Reason: '%s", 1096, STACK_TRACE_WANTED)
```
Comment (eamansour) [inline pkg/errors/errorMessage.go:183]: Missing a closing '

**Review by eamansour — CHANGES_REQUESTED** [review]

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-159-2rsrw for details. If you are unable to do so, please contact a member of the Galasa team.

File: pkg/properties/propertiesSet.go [hunk 6]
```diff
@@ -35,12 +34,9 @@ func SetProperty(
 	// so we assume the user wants to create a new property
 	if err != nil && strings.Contains(err.Error(), "404") {
 		err = createCpsProperty(namespace, name, value, apiServerUrl, console)
-		outputMessage = "Successfully created property '" + name + "' in namespace '" + namespace + "'"
 	}
 
-	if err == nil {
-		console.WriteString(outputMessage)
-	} else {
+	if err != nil {
```
Comment (KirbyKatcher) [inline pkg/properties/propertiesSet.go:39]: Can we get rid of this to avoid duplicate error messages in the cli?

**Review by KirbyKatcher — CHANGES_REQUESTED** [review]

**Comment by galasa-team** [comment]
Build successful

**Review by eamansour — APPROVED** [review]
Looks good!

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**